sdk/FunctionMetadataLoaderExtension/Startup.cs

```diff
@@ -23,17 +23,29 @@ internal class Startup : IWebJobsStartup2, IWebJobsConfigurationStartup
         private const string ExePathPropertyName = "defaultExecutablePath";
         private const string WorkerPathPropertyName = "defaultWorkerPath";
         private const string WorkerRootToken = "{WorkerRoot}";
+        private const string WorkerIndexingPropertyName = "workerIndexing";
 
         private static readonly string _dotnetIsolatedWorkerConfigPath = ConfigurationPath.Combine("languageWorkers", "dotnet-isolated", "workerDirectory");
         private static readonly string _dotnetIsolatedWorkerExePath = ConfigurationPath.Combine("languageWorkers", "dotnet-isolated", ExePathPropertyName);
 
+        private static bool isWorkerIndexingEnabled = false;
+
         public void Configure(WebJobsBuilderContext context, IWebJobsConfigurationBuilder builder)
         {
             string appRootPath = context.ApplicationRootPath;
 
             // We need to adjust the path to the worker exe based on the root, if WorkerRootToken is found.
             WorkerConfigDescription newWorkerDescription = GetUpdatedWorkerDescription(appRootPath);
 
+
+            if (!string.IsNullOrEmpty(newWorkerDescription.EnableWorkerIndexing))
+            {
+                if (newWorkerDescription.EnableWorkerIndexing!.Equals("true", StringComparison.OrdinalIgnoreCase))
```

Comment:
Should we use `bool` as the type for `EnableWorkerIndexing` property in `WorkerConfigDescription` class? The serializer will be able to populate the Boolean value. With that, we probably don't need 41-47 ?